testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/client/OIDCClientRegistrationTest.java

```diff
@@ -321,6 +321,47 @@ public void testMtlsHoKTokenEnabled() throws Exception {
 
     }
 
+    @Test
+    public void testUserInfoEncryptedResponse() throws Exception {
+        OIDCClientRepresentation response = null;
+        OIDCClientRepresentation updated = null;
+        try {
+            // create (no specification)
+            OIDCClientRepresentation clientRep = createRep();
+
+            response = reg.oidc().create(clientRep);
+            Assert.assertEquals(Boolean.FALSE, response.getTlsClientCertificateBoundAccessTokens());
+            Assert.assertNotNull(response.getClientSecret());
```

Comment:
It might be better to delete this two lines.

Reply:
Ok, done